lib/src/toggle-group/ToggleGroup.tsx

```diff
@@ -1,30 +1,30 @@
 import React, { useState } from "react";
 import styled, { ThemeProvider } from "styled-components";
 import { v4 as uuidv4 } from "uuid";
-import PropTypes from "prop-types";
 import { spaces } from "../common/variables.js";
 import useTheme from "../useTheme.js";
+import ToogleGroupPropsType from "./types";
 
 const DxcToggleGroup = ({
   label,
   helperText,
   value,
   onChange,
   disabled = false,
-  options = [],
+  options,
   margin,
   multiple = false,
   tabIndex = 0,
-}) => {
+}: ToogleGroupPropsType): JSX.Element => {
   const colorsTheme = useTheme();
-  const [selectedValue, setSelectedValue] = useState(multiple ? [] : null);
+  const [selectedValue, setSelectedValue] = useState(multiple ? ([] as string[]) : "");
```

Comment:
Is this cast really necessary? I think it would work with just `useState(multiple ? [] : "");`

Reply:
Done.